index.html

```diff
@@ -10,11 +10,6 @@
         <!-- CSS -->
         <link rel="stylesheet" href="css/bootstrap.min.css" />
         
-        <!-- JS -->
```

Comment:
This should be declared at the bottom of the <body> so that the page will load the content and render it before loading and executing javascript, and the page will, as a result, load faster.